### PR TITLE
Migrating Project Manager workflows to sig repo

### PR DESCRIPTION
**Pull request by robhaco**
Signed-off-by: @robhaco

### CODE REVIEW

File: testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+| Step 2 - Gem Details                     | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next                                                                                                                | *   All fields can be edited<br>    *   Gem Display Name<br>    *   Gem System Name<br>    *   Gem Summary<br>    *   Requirements<br>    *   License<br>    *   License URL<br>    *   Origin<br>    *   Origin URL<br>    *   Global Gem Tags<br>    *   Optional Global Gem Tag<br>    *   Optional Global Gem Tag<br>    *   User-defined Gem Tags<br>    *   Gem Location<br>    *   Gem Icon Path<br>    *   Documentation URL<br>*   Global Gem Tags field populates with gem fields |
```
Comment (tkothadev) [inline testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md:16]: Order between gem display name and gem system name need to be swapped.

We currently do not have Origin and Origin URL fields.

In the gem.json file, there is no distinction between User and Global gem tags. We will be just creating a single tag input field, and global gems will be implemented as an auto-completion mechanism. See this PR: https://github.com/o3de/o3de/pull/12501

File: testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+| Gem Details - Global Gem Tag duplication | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next<br>4.  Attempt to enter the same Global Gem Tag into 2 of the Global Gem Tag fields                            | *   An error displays notifying the user that duplicates cannot be used for global gem tags                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (tkothadev) [inline testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md:17]: Although we do not have global gem tags, in the PR for the gem tag field, we actually prevent users from creating duplicate tags in the first place:

https://github.com/o3de/o3de/pull/12501

File: testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+| Step 2 - Required Fields not satisfied   | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next<br>4.  On step 2 page, click next without entering information into some of the required fields (marked by \*) | *   User is unable to progress without required fields and a visual indicator suggests what is missing<br>*   Required Fields:<br>    *   Gem Display Name<br>    *   Gem System Name<br>        *   Alpha numeric requirement<br>        *   Unique error for alpha numeric<br>    *   License<br>    *   Global Gem Tag                                                                                                                                                                   |
```
Comment (tkothadev) [inline testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md:18]: Should a tags field be required? I think it's reasonable to have a gem with initially 0 tags to it, especially if users are starting out.

File: testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+| Step 3 - Creator Details                 | 1.  Launch the Create a Gem experience<br>2.  Select a Gem Type/Template<br>3.  Left Click Next<br>4.  Enter Details for Gem<br>5.  Left Click Next                                                            | *   User is unable to progress without required fields and a visual indicator suggests what is missing<br>*   Required Fields:<br>    *   Creator Name<br>    *   Repository URL                                                                                                                                                                                                                                                                                                            |
```
Comment (tkothadev) [inline testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md:19]: Only creator name is required. It's possible the gem is just created for the local drive, so the existing path info would cover that case.

**Review by AMZN-Phil — APPROVED** [review]

**Pull request merged**